bucket/heroku-cli.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": "7.26.2",
+    "homepage": "https://devcenter.heroku.com/articles/heroku-cli",
+    "description": "An essential part of using Heroku, which can perform nearly any Heroku-related task right from your terminal.",
+    "license": "ISC",
+    "architecture": {
+        "64bit": {
+            "url": "https://cli-assets.heroku.com/heroku-win32-x64.tar.xz#/dl",
+            "hash": "b552e8c121df572b25a7c96e7272c6eeacdfa49dac0af085ee78b03710b28739"
+        },
+        "32bit": {
+            "url": "https://cli-assets.heroku.com/heroku-win32-x86.tar.xz#/dl",
+            "hash": "b9cab430271936f58f8fc5b4da663307ad6292ab10b6d5a24dfbe642c686518c"
+        }
+    },
+    "depends":"sudo",
+    "bin": "bin/heroku.cmd",
+    "checkver": {
+        "url": "https://cli-assets.heroku.com/version",
+        "jsonpath": "$.version"
+    },
+    "pre_install": [
+        "& (scoop which 7z) x \"$dir\\dl\" -o\"$dir\" | Out-Null",
+        "Invoke-ExternalCommand -FilePath (scoop which 7z) -ArgumentList \"x `\"$dir\\dl~`\" -o`\"$dir\\dlo`\" -bso0 -bd\" -RunAs | Out-Null",
+        "Move-Item \"$dir\\dlo\\heroku\\*\" \"$dir\"",
+        "Remove-Item \"$dir\\dl*\" -Recurse"
+    ],
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://cli-assets.heroku.com/heroku-win32-x64.tar.xz#/dl",
```

Comment:
Why there is rename without extension??

Reply:
Because it need to be decompressed with privilege. So I have to rename it to bypass auto decompress.